docs/criteria/connectors/rc.md

```diff
@@ -136,6 +140,7 @@ Indexes are not required for test coverage, but can be introduced if required fo
 - [ ] A test script exists that can load TPC-H data at the [designated scale factor](#rc-release-criteria) into this connector.
 - [ ] The connector can load TPC-H at the [designated scale factor](#rc-release-criteria), and can run all queries with no [Major Bugs](../definitions.md).
 - [ ] All [Minor Bugs](../definitions.md) for TPC-H are raised as issues.
+- [ ] TPC-H queries pass with a success rate equal or greater than TPC-H execution on Datafusion.
```

Comment:
It is good to elaborate more on  `success rate`  -  does it mean percentage of passed queries? for ex Postgres fails on Q1 and Q2, but works for Q3 and Q4 and Arrow fails for Q3 and Q4 and passes Q1 and Q2 - does this case satisfies `success rate` criteria?

Reply:
I've changed this to call out comparing against the individual query executions on Datafusion